.ci/runtimes.txt

```diff
@@ -0,0 +1,418 @@
+unit	spec/01-unit/29-lua_cjson_large_str_spec.lua	2.97090870350005
```

Comment:
as this file changes every week, it may make our git repo size grow quick. would it be possible to host this file (and the workflow that generates it too) in a seperate repo?

Reply:
We discussed this in the PR review meeting today and decided that the repo size increase is acceptable for now.  We may need to revisit in the future.

Reply:
It was not really for the size increase, but for the review and signature requirements that exist in the Kong repo that made us switch to using a separate repository.